.circleci/continue_config.yml

```diff
@@ -391,7 +391,6 @@ workflows:
               extras: [ "none" ]
           pytest-group: dataset
           enabled: true
-          parallelism: 2
```

Comment:
https://github.com/kolenaIO/kolena/pull/414/commits/37ff420bdfb4cbba8765032b1eda00e17b6eb915 doesn't fix the issues, we create a dummy test under dataset to bypass the parallelism is lower than the total of tests issue, which will cause one box to run all integration tests and it takes forever

Reply:
for unknown reasons, added `parallelism: 2` caused two `integration-test-dataset-3.9.18-none` jobs were triggered: `integration-test-dataset-3.9.18-none-1` and `integration-test-dataset-3.9.18-none-2`. and one of them has parallelism of 2 and another one has parallelism of 3